singlevenue/src/main/java/com/mappedin/examples/singlevenue/MainActivity.java

```diff
@@ -536,6 +621,10 @@ private void startNavigation(Polygon from, Polygon to) {
         walkingButton.setVisibility(View.VISIBLE);
         Directions directions =
                 from.directionsTo(activeVenue, to, from.getLocations()[0], to.getLocations()[0], accessibleDirections);
+
+        levelChangeDownButton.setEnabled(false);
```

Comment:
I dont think we should be preventing level changes during navigation. Sometimes navigation could be on two floors.

Reply:
I disabled user level navigation between floors when the user clicks "Go" because it causes some weird things to happen to the generated path if the user switches floors with the animation is happening. The camera still automatically tracks the path taken when the user clicks "Go", and will automatically change floors when a vortex is taken.